packages/generate/tsconfig.json

```diff
@@ -1,21 +1,9 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    // "target": "es2017",
-    // "strictPropertyInitialization": false,
-    // "baseUrl": ".",
```

Comment:
I'm not a big fan of commented out code, so I just removed all this extra stuff 🤷